Packs/Code42/ReleaseNotes/6_0_5.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Code42 Event Collector
+
+- Updated the Code42 Event Collector integration to fetch File Events correctly.
```

Comment:
Need to indicate all changes we made. like the new parameter, the lookback, the new time parameter.. etc.